### PR TITLE
Intersect. Theory: Code cleanup, correction, new example

### PR DESCRIPTION
**Pull request by wdecker**

### CODE REVIEW

File: experimental/IntersectionTheory/docs/src/intro.md [hunk 1]
```diff
@@ -6,12 +6,18 @@ CurrentModule = Oscar
...
-- give examples which illustrate how intersection theory is used to
-solve problems from enumerative geometry.
+- give examples which illustrate how intersection theory is used to solve problems from enumerative geometry.
 
 !!! note
-    Making use of OSCAR, a first version of what we present here was
-    written by Jeiao Song as a julia package.
+    Making use of OSCAR, a first version of what we present here was written by Jeiao Song as a julia package.
+    This package was "heavily inspired by the Macaulay2 package Schubert2 and the Sage library Chow. Some
+	functionalities from  [the Sage library] Schubert3 are also implemented."
```
Comment (lgoettgens) [inline experimental/IntersectionTheory/docs/src/intro.md]:
```suggestion
    functionalities from [the Sage library] Schubert3 are also implemented."
```
whitespace fixes

File: experimental/IntersectionTheory/docs/src/intro.md [hunk 2]
```diff
@@ -6,12 +6,18 @@ CurrentModule = Oscar
...
+
+!!! note
+    Schubert2 was written by Daniel R. Grayson, Michael E. Stillman, Stein A. Strømme, David Eisenbud, and Charley Crissman
+    while Chow is due to Manfred Lehn and Christoph Sorger. Schubert3  as well as the Singular library schubert.lib were
+	written by Dang Tuan Hiep. The basis for all this work, including ours, is the Maple package Schubert written
+	by Sheldon Katz and Stein A. Strømme.
```
Comment (lgoettgens) [inline experimental/IntersectionTheory/docs/src/intro.md]:
```suggestion
    written by Dang Tuan Hiep. The basis for all this work, including ours, is the Maple package Schubert written
    by Sheldon Katz and Stein A. Strømme.
```
more whitespace

File: experimental/IntersectionTheory/src/Main.jl [hunk 3]
```diff
@@ -1295,7 +1295,7 @@ function schur_functor(F::AbstractBundle, λ::Partition)
   λ = conjugate(λ)
   X = F.parent
   w = _wedge(sum(λ), chern_character(F))
-  S, ei = polynomial_ring(QQ, "e#" => 1:length(w))
+  S, ei = polynomial_ring(QQ, ["e$i" for i in 1:length(w)])
```
Comment (lgoettgens) [inline experimental/IntersectionTheory/src/Main.jl]:
```suggestion
  S, ei = polynomial_ring(QQ, "e#" => 1:length(w))
```
this produces the same, but is more efficient

File: experimental/IntersectionTheory/src/blowup.jl [hunk 4]
```diff
@@ -14,24 +14,62 @@ $A^r \rightarrow A^s\rightarrow B \rightarrow 0$
...
+- `sect` is a function which gives rise to a section of the augmentation map $ A^s\rightarrow B$.
 
 !!! note
     The finiteness condition on `F` is checked by the function.
 
+!!! note
+    The function is implemented so that the last element of `gs` is `one(B)`.
+
 # Examples
+```
```
Comment (lgoettgens) [inline experimental/IntersectionTheory/src/blowup.jl]:
```suggestion
```jldoctest
```
or is there any good reason to not do this?

File: experimental/IntersectionTheory/src/blowup.jl [hunk 5]
```diff
@@ -129,58 +166,53 @@ function  present_finite_extension_ring(F::Oscar.AffAlgHom)
...
-	       y -= q * gensB_lift[i]
-	     end; ans)
+
+  sect = x -> (y = reduce(BRtoR(x), gens(V));
+	      ans = elem_type(AR)[];
+	      for i in 1:g
+	        q = div(y, gs_lift[i])
+	        push!(ans, RtoAR(q))
+	        y -= q * gs_lift[i]
+	      end; ans)
```
Comment (lgoettgens) [inline experimental/IntersectionTheory/src/blowup.jl]: there are some more tabs in the above paragraph

File: experimental/IntersectionTheory/src/blowup.jl [hunk 6]
```diff
@@ -229,60 +261,85 @@ julia> integral((6*HBl-2*e)^5)
...
+  AˣBl, _ = quo(RBl, Rels)
   Bl = abstract_variety(Y.dim, AˣBl)
 
-# Bl being constructed, we add the morphisms f, g, and j
-  RBltoRY = hom(RBl, RY, vcat(repeat([RY()], n), gens(RY)))
+# Bl being constructed, we set up the morphisms f, g, and j
+
+  RBltoRY = hom(RBl, RY, vcat(repeat([RY()], ngs), gens(RY)))
   fₓ = x -> (xf = simplify(x).f;
 	     Y(RBltoRY(xf));)
```
Comment (lgoettgens) [inline experimental/IntersectionTheory/src/blowup.jl]: more tabs
```suggestion
         Y(RBltoRY(xf));)
```

File: experimental/IntersectionTheory/src/blowup.jl [hunk 7]
```diff
@@ -293,14 +350,14 @@ function blowup(i::AbstractVarietyMap; symbol::String = "e")
...
   
   # pushforward of j: write as a polynomial in ζ, and compute term by term
   RX = base_ring(X.ring)
   RPNtoRX = hom(base_ring(PN.ring), RX, pushfirst!(gens(RX), RX()))
   jₓ = x -> (xf = simplify(x).f;
 	     RX = base_ring(X.ring); ans = RBl();
-	     for k in d-1:-1:0
+	     for k in rN-1:-1:0
 	       q = div(xf, ζ.f^k)
 	       ans += jₓgˣ(X(RPNtoRX(q))) * (-E[end])^k
```
Comment (lgoettgens) [inline experimental/IntersectionTheory/src/blowup.jl]: more tabs in this paragraph

**Review by lgoettgens — COMMENTED** [review]
some misc comments, mostly whitespace

File: experimental/IntersectionTheory/src/blowup.jl (hunk 5 above)
Comment (wdecker) [reply experimental/IntersectionTheory/src/blowup.jl]: @lgoettgens please feel free to remove the tabs

File: experimental/IntersectionTheory/src/blowup.jl (hunk 7 above)
Comment (wdecker) [reply experimental/IntersectionTheory/src/blowup.jl]: @lgoettgens please, as above with regard to the tabs.
thanks for catching the typo: can you please make the replacement

File: experimental/IntersectionTheory/src/blowup.jl (hunk 5 above)
Comment (lgoettgens) [reply experimental/IntersectionTheory/src/blowup.jl]: done (in https://github.com/oscar-system/Oscar.jl/pull/4180/commits/7435872d4eae7b0e667537b5c917493da60f945f)

**Pull request merged**